commit d5cb3ba0911ed6f22933b4c6f713e41f3f6336de
Author: Alexandre Belloni <alexandre.belloni@free-electrons.com>
Date:   Thu Jan 19 01:46:58 2017 +0100

    ARM: dts: at91: Enable DMA on sama5d4_xplained console
    
    commit ef8d02d4a2c36f7a93e74c95a9c419353b310117 upstream.
    
    Enable DMA on usart3 to get a more reliable console. This is especially
    useful for automation and kernelci were a kernel with PROVE_LOCKING enabled
    is quite susceptible to character loss, resulting in tests failure.
    
    Acked-by: Nicolas Ferre <nicolas.ferre@atmel.com>
    Signed-off-by: Alexandre Belloni <alexandre.belloni@free-electrons.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1230_d5cb3ba0911ed6f22933b4c6f713e41f3f6336de.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1230_d5cb3ba0911ed6f22933b4c6f713e41f3f6336de.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
